packages/connect-query/package.json

```diff
@@ -22,16 +21,13 @@
   "type": "module",
   "sideEffects": false,
   "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
I think the most simple change would be to delete node from exports, so that we end up with:

```
"exports": {
    ".": {
      "module": "./dist/esm/index.js",
      "import": "./dist/proxy/index.js",
      "require": "./dist/cjs/index.js"
    }
}
```

Note that we're using the module export instead of adding the module field.

It's a good idea to match the upstream package here, but since connect-es and protobuf-es are involved as well, it's better we stick to incremental changes that we can cover in our tests in examples-es.

Note that this change will potentially re-introduce dual package issues around ConnectError. @smaye81 is looking into making the internal usage in ConnectError resilient against dual package issues, which is ultimately the better way forward than the ESM wrappers.

Reply:
The additional `module` field was just there to mimic react-query, since it doesn't seem to make a difference with nextjs. The problem is largely that this bug triggers if `module` and `import` don't actually match. So basically, we could do something like:

```ts
"exports": {
    ".": {
      "module": "./dist/esm/index.js",
      "import": "./dist/esm/index.js",
      "require": "./dist/cjs/index.js"
    }
}
```

And it'll work, though now the `module` designation seems redundant.

Reply:
I thought so too, but @smaye81's recent tests in https://github.com/connectrpc/examples-es/pull/1149 proved otherwise.

Reply:
Sounds good... well, not good, just confusing, but I'll keep module.